### ISSUE TITLE
Website Improvement

### ISSUE THREAD

**Issue by reveurguy** [kubevirt/kubevirt.github.io#883]
**Is this a BUG REPORT or FEATURE REQUEST?**:
The website can be improved visually which would make it look and work better. I would like to work on this and re-design the website part by part to improve it.

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What happened**:

**What you expected to happen**:

**Anything else we need to know?**:

> URL where the problem can be found ...
> If the issue is with a lab, please provide information about your environment, platform, versions, ...

**Comment by cwilkers** [comment]
Interesting, most of the website features should be straightforward to figure out; reach out if you need help understanding the CI/CD process we have here.

**Comment by aburdenthehand** [comment]
@reveurguy Thanks for the offer! Does this include size and placement of the vendor/integration/end user images on the main page?

**Comment by reveurguy** [comment]
Yeah, I would like to work on the complete website, starting with the the homepage. I am running a bit busy for the last few weeks, will start from tomorrow and then we can discuss it going forward.
Will open a PR tomorrow with some initial changes

**Comment by FrankyV52** [comment]
Hi, my name is Frankie and I am a senior at UT Austin studying computer science. My group and I are interested in contributing to this issue. Is there anything in particular that we could get started with?

**Comment by cwilkers** [comment]
Hello @reveurguy and @FrankyV52 ,

One important design consideration is to ensure that changes to this repo are reflected also in the kubevirt/user-guide repo because the sites work seamlessly together. Right now there are some slight differences.

One nice feature request would be  a banner for events (like KubeVirt Summit this week, Wednesday and Thursday!) that currently we only really announce via a blog.

I will be doing a tutorial session this Thursday for the KubeVirt Summit on getting started with updating our documentation. It's called Don't Knock the Docs: Contributing Documentation to the KubeVirt Project. Attendance to the conference is free and you can get started here: https://community.cncf.io/events/details/cncf-kubevirt-community-presents-kubevirt-summit-2023/

Note the times are UTC, so my session is noon CDT (I'm in Houston btw)

**Comment by reveurguy** [comment]
Hello @cwilkers . Sorry for so much delay, i got occupied with university exams and other stuff and completely forgot about this involvement.
I am currently giving my exams, so I will surely get started with my suggestions from around mid-June.

I also wanted to discuss if changing the website tech-stack would be a viable option. I know there would be a lot of stuff to change and manage with that. But I wanted to discuss whether moving the project to use React or Nextjs frameworks is something the maintainers are willing to discuss about.
There would be many improvements to the development process and managing as well if a change is made to a modern and latest framework.
I am willing to discuss upon this further.

If the change doesn't seem suitable, that's fine as well. I'll get started with the current environment only but from mid-June.

**Comment by reveurguy** [comment]
hello @cwilkers any updates on this?
